driver/level2/symv_thread.c

```diff
@@ -166,7 +166,11 @@ int CNAME(BLASLONG m, FLOAT *alpha, FLOAT *a, BLASLONG lda, FLOAT *x, BLASLONG i
     if (nthreads - num_cpu > 1) {
 
       double di = (double)i;
-      width = ((BLASLONG)(sqrt(di * di + dnum) - di) + mask) & ~mask;
+      if (di * di - dnum > 0) {
+        width = ((BLASLONG)(-sqrt(di * di - dnum) + di) + mask) & ~mask;
```

Comment:
I'm not sure I understand why there's a negative here.

Reply:
Oh, man, I screwed this one up badly :(, sorry, I guess have to revert...

Reply:
I'm worse, I signed off on what I thought I was seeing.

Reply:
Btw, can `di*di + dnum` be negative ?

Reply:
Only when the number of threads is negative methinks (lets call the associated hardware a disputer)